bin/opencollective.js

```diff
@@ -24,10 +24,15 @@ function printBadge() {
 	console.log("\n");
 	print(`${chalk.bold("Thanks for using")} ${chalk.bold.blue("Webpack!")}`);
 	print(`Please consider donating to our ${chalk.bold.blue("Open Collective")}`);
-	print("to help us maintain this package.");
+	print("to help us to maintain this package.");
```

Comment:
```suggestion
	print("to help us maintain this package.");
```

Reply:
Thanks, see [here](https://github.com/webpack/webpack-cli/pull/863#discussion_r279170165)

Reply:
To me w/o `to` (that I suggested to remove) makes more sense still as we have already used `to` before `help` and adding another `to` is redundant. Rest on you @misterdev :)

Reply:
Changed, I prefer your version too :)